src/bci_build/package/kubectl.py

```diff
@@ -0,0 +1,24 @@
+"""Kubectl Client BCI container"""
+
+from bci_build.os_version import OsVersion
+from bci_build.package import ApplicationStackContainer
+from bci_build.package.versions import get_pkg_version
+
+KUBECTL_CONTAINERS = []
+KUBECTL_SUPPORTED_OS_VERSIONS = [
+    OsVersion.TUMBLEWEED,
+]
+
+for os_version in KUBECTL_SUPPORTED_OS_VERSIONS:
+    KUBECTL_CONTAINERS.append(
+        ApplicationStackContainer(
+            name="kubectl",
+            os_version=os_version,
+            version=get_pkg_version("kubernetes", os_version),
+            license="MPL-2.0",
+            is_latest=True,
```

Comment:
only in "os_version in CAN_BE_LATEST_OS_VERSION"

maybe you could simply copy&paste https://github.com/SUSE/BCI-dockerfile-generator/blob/main/src/bci_build/package/cosign.py and simply search and replace "cosign" with "kubernetes-client" to get started?